src/modules/Dropdown/Dropdown.js

```diff
@@ -158,62 +155,69 @@ export default class Dropdown extends Component {
-    basic: PropTypes.bool,
-
-    /** Format the Dropdown to appear as a button. */
-    button: PropTypes.bool,
-
-    /** Additional classes. */
-    className: PropTypes.string,
-
-    /** Format the dropdown to only take up as much width as needed. */
-    compact: PropTypes.bool,
-
-    /** Format the dropdown to only take up as much width as possible. */
-    fluid: PropTypes.bool,
-
-    /** Display the menu as detached from the Dropdown. */
-    floating: PropTypes.bool,
-
-    /** A dropdown menu can contain a header. */
-    header: PropTypes.node,
+    /** Controls whether or not the dropdown menu is displayed. */
+    open: PropTypes.bool,
 
-    inline: PropTypes.bool,
-    labeled: PropTypes.bool,
-    // linkItem: PropTypes.bool,
+    /** Array of Dropdown.Item props e.g. `{ text: '', value: '' }` */
+    options: customPropTypes.every([
+      customPropTypes.disallow(['children']),
+      PropTypes.arrayOf(PropTypes.shape(DropdownItem.propTypes)),
+    ]),
 
-    /** Allow selecting multiple options. */
-    multiple: PropTypes.bool,
+    /** Placeholder text. */
+    placeholder: PropTypes.string,
 
-    /** Use a detached menu that is pointing to the Dropdown. */
+    /** A dropdown can be formatted so that its menu is pointing. */
     pointing: PropTypes.oneOfType([
       PropTypes.bool,
       PropTypes.oneOf(_meta.props.pointing),
     ]),
 
-    /** The text displayed in the dropdown, usually for the active item. */
-    text: PropTypes.string,
+    /** A dropdown can have its menu scroll. */
+    scrolling: PropTypes.bool,
+
+    /**
+     * A selection dropdown can allow a user to search through a large list of choices.
+     * Pass a function here to replace the default search.
+     */
+    search: PropTypes.oneOfType([
+      PropTypes.bool,
+      PropTypes.func,
+    ]),
 
     // TODO 'searchInMenu' or 'search='in menu' or ???  How to handle this markup and functionality?
 
-    /** Behave as an html select. */
+    /** A dropdown can be used to select between choices in a form. */
     selection: customPropTypes.every([
       customPropTypes.disallow(['children']),
       customPropTypes.demand(['options']),
       PropTypes.bool,
     ]),
+
+    /** Define whether the highlighted item should be selected on blur. */
+    selectOnBlur: PropTypes.bool,
+
+    /** A simple dropdown can open without Javascript. */
     simple: PropTypes.bool,
 
-    loading: PropTypes.bool,
-    error: PropTypes.bool,
-    disabled: PropTypes.bool,
+    /** The text displayed in the dropdown, usually for the active item. */
+    text: PropTypes.string,
 
-    scrolling: PropTypes.bool,
+    /** Custom element to trigger the menu to become visible. Takes place of 'text'. */
+    trigger: customPropTypes.every([
+      customPropTypes.disallow(['selection', 'text']),
+      PropTypes.node,
+    ]),
+
+    /** Current value or value array if multiple. Creates a controlled component. */
+    value: PropTypes.oneOfType([
```

Comment:
No functional changes took place. I sorted the props to be in alphabetical order and updated the comment descriptions to match SUI core where relevant.